experiments/functional/zfs-LocalPV/zfspv-clone/test_vars.yml

```diff
@@ -0,0 +1,21 @@
+test_name: zfspv-clone
+
+app_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+
+storage_class: "{{ lookup('env','STORAGE_CLASS') }}"
+
+snapshot_class: "{{ lookup('env','SNAPSHOT_CLASS') }}"
```

Comment:
Do we really need snapshot_class

Reply:
No, we don't need...
Removed this unused variable.
Thanks ! @somesh2905